mteb/abstasks/AbsTaskRetrieval.py

```diff
@@ -376,44 +376,78 @@ def _add_main_score(self, scores: ScoresDict) -> None:
     def calculate_metadata_metrics(self) -> None:
         self.load_data()
 
+        all_details = {}
         for split in self.metadata_dict["eval_splits"]:
+            all_details[split] = {}
             if self.is_multilingual:
                 for lang in self.relevant_docs.keys():
-                    process_language(
+                    print(split, lang)
```

Comment:
Was this intentional or for debugging purposes?

Reply:
There were the other print statements, so I wasn't sure -- should I keep them in (and make them nicer) or take them out?

Reply:
Here we often see multilingual datasets with a huge number of languages. Showing process would be nice to have. Actually instead of print, maybe a progress bar could be helpful?
```
for lang in tqdm(self.relevant_docs.keys()):
    ...
```

Reply:
I like the tqdm bars, but realized when I switched it to that locally that the results print statements are then unattached to what split they came from.  So I will leave the print statements and add the TQDM bars